site/index.html

```diff
@@ -14,10 +14,19 @@
 
 <div id="app">
   <aside id="sidebar">
-    <div class="sidebar-heading">
-      <b>Channels</b>
-      <button class="btn" id="create-channel">New</button>
-    </div>
+    <section class="sidebar-section sidebar-section-light">
+      <div class="sidebar-heading">
+        <div id="server-status">Connecting...</div>
```

Comment:
Should this be "not connected" in the HTML, then set to "Connecting..." by JS?

Reply:
I guess. The result is the same, however.